src/package_io/parse_input.star

```diff
@@ -101,9 +101,6 @@ def parse_input(input_args):
 	if result["network_params"]["genesis_delay"] == 0:
 		fail("genesis_delay is 0 needs to be > 0 ")
 
-	if result["network_params"]["capella_fork_epoch"] == 0:
```

Comment:
can we add an if condition that checks if mev is enabled and complain if capella epoch is less than 1